packages/create-vite/template-solid-ts/src/App.tsx

```diff
@@ -0,0 +1,34 @@
+import { useSignal } from 'solid-js'
+import solidLogo from './assets/solid.svg'
+import './App.css'
+
+function App() {
+  const [count, setCount] = useSignal(0)
+
+  return (
+    <div className="App">
+      <div>
+        <a href="https://vitejs.dev" target="_blank">
+          <img src="/vite.svg" className="logo" alt="Vite logo" />
```

Comment:
Recently we changed to import `/vite.svg` like so

https://github.com/vitejs/vite/blob/c49af2340b026f55b5855343c59272a9a733bd2f/packages/create-vite/template-preact-ts/src/app.tsx#L1-L13

Solid also uses `class` instead of `className`